charts/flink-tools/templates/FlinkCluster.yaml

```diff
@@ -22,12 +22,8 @@ spec:
         requests:
           storage: {{ .Values.localStorage.size | quote }}
       storageClassName: standard
-{{- if .Values.enableExtraVolume }}
   volumes:
-    - name: extra-volume
-      persistentVolumeClaim:
-        claimName: flink-tools-nfs-pvc
-{{- end }}
+  {{ toYaml .Values.volumes | indent 4 }}
```

Comment:
Check your spacing here. I believe that this needs to start at the first column, like line 33.

Reply:
Fixed